tests/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+---
+version: '3.4'
+
+services:
+
+    aiidalab:
+        image: aiidalab/full-stack:edge
+        environment:
+            RMQHOST: messaging
+            TZ: Europe/Zurich
+            DOCKER_STACKS_JUPYTER_CMD: notebook
+            SETUP_DEFAULT_AIIDA_PROFILE: 'true'
+            AIIDALAB_DEFAULT_APPS: ''
+            JUPYTER_TOKEN: ${JUPYTER_TOKEN}
+        volumes:
+            - ..:/home/jovyan/apps/aiidalab-widgets-base
+        ports:
+            - 8998:8888
```

Comment:
Any reason not to use `8888:8888`?

Reply:
Ha nice catch. Only personal reason since the `8888` of my workstation is occupied by some wired raid service. Will change it back to `8888`.

Reply:
Hi @danielhollas, I am thinking maybe it is better to keep the host port other than `8888`, since `8888` is the default port for jupyter and high possibility that already been used by user. This docker-compose.yml will only be used for test purposes and is triggered by `pytest`, it more convenient if it does not conflict with `8888`.